src/index.js

```diff
@@ -3,15 +3,19 @@ import ReactDOM from "react-dom";
 import App from "./App";
 import { makeServer } from "./server";
 import {BrowserRouter} from 'react-router-dom'
+import { Provider } from "./Context/context";
+
 
 // Call make Server
 makeServer();
 
 ReactDOM.render(
   <React.StrictMode>
+  <Provider>
```

Comment:
here, I think you can provide a better name like if its product provider then `ProductProvider`

Reply:
Will change that.